uniswap-v2-core/contracts/CalcFee.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: UNLICENSE// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.7.0 <0.9.0;
+
+contract CalcFee {
```

Comment:
Where do you use this code?

Reply:
This was an example file to let people deploy and test the minting function in isolation. This code made it into UniswapV2Pair.sol under `_calcFee`